pkg/controller/replicas.go

```diff
@@ -0,0 +1,160 @@
+package controller
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	v1 "k8s.io/api/apps/v1"
+
+	"github.com/grafana/rollout-operator/pkg/admission"
+)
+
+type ScaleAction int
+
+const (
+	NoChange ScaleAction = iota
+	ScaleUp
+	ScaleDown
+)
```

Comment:
Nit: not sure if these are going to stay (see Marco's comment below), but if they do, these don't need to be exported.

Reply:
Removed in 47883c74c59e28e03da6587b260af0b3cf024b70